### DISCUSSION TITLE
.fig file for Figma not received

### DISCUSSION THREAD

**Discussion by lholeman5** [Laravel-Backpack/community-forum#633] [General] [answered]
Hi,
I purchased the full package and the documents say that I am to receive a .fig file.  I am not seeing it anywhere.  I set the account up but would like to move to the next step.
Thanks,
Larry

**Comment by promatik** [comment]
Hi @lholeman5!
Sorry for the trouble, we're on it 👌
@tabacitu will answer you as soon as possible.

**Comment by tabacitu** [comment, marked as answer]
Hi @lholeman5 ,

That is correct. If you've purchased our EVERYTHING bundle, or the Figma Template individually, you do have access to the Figma template. It doesn't get downloaded automatically though. You have to download it from its page: https://backpackforlaravel.com/products/figma-template

![CleanShot 2023-08-14 at 13 09 46](https://github.com/Laravel-Backpack/community-forum/assets/1032474/e54c80a5-7b9f-4bd0-9dc8-6e306822d6eb)

Hope it helps.
Cheers!
